backend/analysis_framework/serializers.py

```diff
@@ -0,0 +1,62 @@
+from rest_framework import serializers
+from user_resource.serializers import UserResourceSerializer
+from .models import (
+    AnalysisFramework, Widget, Filter, Exportable
+)
+
+
+class AnalysisFrameworkSerializer(UserResourceSerializer):
+    """
+    Analysis Framework Model Serializer
+    """
+    class Meta:
+        model = AnalysisFramework
+        fields = ('__all__')
+
+
+class WidgetSerializer(serializers.ModelSerializer):
+    """
+    Widget Model Serializer
+    """
+
+    class Meta:
+        model = Widget
+        fields = ('__all__')
+
+    # Validations
+    def validate_region(self, widget):
```

Comment:
We need to validate the field *analysis_framework*. Only users who can modify the analysis framework can add widgets to it.

This should be `validate_analysis_framework(self, analysis_framework)`.

Raise error 'Invalid analysis framewok'